lib/plug/ssl.ex

```diff
@@ -59,6 +59,26 @@ defmodule Plug.SSL do
   HSTS expects the port to be 443 for SSL. If you are not using HSTS and
   want to redirect to HTTPS on another port, you can sneak it alongside
   the host, for example: `host: "example.com:443"`.
+
+  ## Excluded hosts tuple
+
+  In case list needs to be populated during runtime, tuple
+  `{module, function, args}` can be passed to be invoked when checking
+  whether to redirect. The specified function needs to return the list.
+
+  For example, `config/prod.exs` of a Phoenix app can contain:
+
+      config :sample_app, SampleAppWeb.Endpoint,
+        force_ssl: [
+          rewrite_on: [:x_forwarded_proto],
+          exclude: {Application, :get_env, [:sample_app, :ssl_excluded_hosts]}
```

Comment:
Sorry, I think there was some confusion. The tuple shouldn't return a list, the tuple should receive the current host and return if it is excluded or not. In other words, the while logic should be in the given tuple.

Reply:
Oh ok, makes sense.